### PR TITLE
New post on Testing mobile apps with Calabash, by author aikhelis

### PR DESCRIPTION
**Pull request by aikhelis**

### CODE REVIEW

**Comment by aikhelis** [comment]
@jphastings @gregbeech @rahilb - I know it is a little bit late, but wanted to contribute to the Tech legacy with a long waited article :) Appreciate a review.

**Comment by jphastings** [comment]
Looks good to me @aikhelis, any thoughts @gregbeech, or shall we merge?

File: _posts/2015-01-21-oo-page-models-for-calabash.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+  class SignInPage < PageObjectModel::Page
+    trait "TextView marked:'Sign in'"
+    element :email_field, "* id:'edittext_email'"
+    element :password_field, "* id:'edittext_password'"
+    element :signin_button, "* id:'button_signin'"
+    element :signin_label, "* text:'Sign in'"
+
+    def submit_sign_in_details(user_name, password)
+      clear_text(email_field)
+      enter_text(email_field, user_name)
```
Comment (gregbeech) [inline _posts/2015-01-21-oo-page-models-for-calabash.md]: Didn't I see a `set` method above which wrapped this clear/enter paradigm?

Comment (aikhelis) [inline _posts/2015-01-21-oo-page-models-for-calabash.md]: This last section in the article shows how the same page model and step definitions could be implemented without our OO adaptor. #set method does not exist in Calabash, it's one of the operations we added to the list of standard operations.

Comment (aikhelis) [inline _posts/2015-01-21-oo-page-models-for-calabash.md]: I should probably make it clear before this code block

**Comment by gregbeech** [comment]
Content seems good to me. Purely stylistically I find it a bit "bullet pointy" and would probably find it easier to read if the text was more flowing, but happy to merge.

**Comment by aikhelis** [comment]
Thanks @gregbeech and @jphastings - makes sense. I can work on the narration in a separate pull request, if this is good to be published alongside yours? Will need to find a bit of time.

**Pull request merged**